app/helpers/format-changes.ts

```diff
@@ -20,7 +20,7 @@ export function formatChanges(
     .filter(({ type, before, after }) =>
       type === "checkboxes"
         ? !arrayEquals(before as string[], after as string[])
-        : before !== after
+        : before !== after,
     )
-    .filter(({ before, after }) => before || after);
+    .filter(({ before, after }) => before !== null || after !== null);
```

Comment:
it's ok if before or after is falsy, just want to makes ure they're not both null